Project.cmake

```diff
@@ -36,6 +39,9 @@ if(WIN32)
 
 	# Install binaries in the root of the install directory
 	set(INSTALL_BIN_DIR "/" CACHE PATH "Where to place binaries in the instalation")
+elseif(APPLE)
+	set(INSTALL_BIN_DIR "MacOS" CACHE PATH "")
+	set(PROJECT_BINARY_DIR ${CMAKE_INSTALL_PREFIX})
```

Comment:
`set(PROJECT_BINARY_DIR ${CMAKE_INSTALL_PREFIX})` Seems like a real bad idea :D

`set(INSTALL_BIN_DIR "MacOS" CACHE PATH "")` can be just `set(INSTALL_BIN_DIR "MacOS")` since it doesn't effect anything on OSX